Makefile

```diff
@@ -108,11 +108,21 @@ build-release-tarballs:
 test-release:
 	cargo test --workspace --release --exclude ef_tests --exclude beacon_chain --exclude slasher
 
+# Runs the full workspace tests in **release**, without downloading any additional
+# test vectors, using nextest.
+nextest-release:
+	cargo nextest run --workspace --release --exclude ef_tests --exclude beacon_chain --exclude slasher
```

Comment:
Just reading through the docs in the default nextest config - for CI, it might be useful to run with `--no-fail-fast` and `--failure-output immediate-final`?

https://github.com/sigp/lighthouse/blob/ebd97bfa8390788dde4cd13382eca1fee4411c6a/.config/nextest.toml#L66-L68

https://github.com/sigp/lighthouse/blob/ebd97bfa8390788dde4cd13382eca1fee4411c6a/.config/nextest.toml#L49-L60

Reply:
I think we might ideally want `fail-fast` for Bors and full testing for PRs. If we had to choose, I'd say fail-fast, because then we get feedback the quickest and free up the runner.

Reply:
yeah good point  👍  given the amount of time our CI takes 🐢  (but soon 🐰!! )

Reply:
Yeah. I saw this too. I thought we wanted to end tests straight up and exit, for resource use. 

Often if there is a bug, usually i can find it without knowing all future test results